src/ucp/wireup/signaling_ep.c

```diff
@@ -103,6 +106,9 @@ ucp_signaling_ep_tag_eager_short(uct_ep_h ep, uct_tag_t tag, const void *data,
     ctx.payload = data;
     ctx.length  = length;
 
+    ucs_assert(ucp_memory_type_detect(proxy_ep->ucp_ep->worker->context,
```

Comment:
common macro ucp_assert_memtype(context, buffer length, mem_type)

Reply:
done, added `UCP_ASSERT_MEMTYPE` macro to ucp/core/ucp_context.h